dsc/examples/PSConfEU_2024_Demo.md

```diff
@@ -0,0 +1,103 @@
+# PSConfEU 2024 Demos
+
+## DSC Command-line
+
+### List DSC resources
+
+- List DSC resources
+
+```powershell
+dsc resource list
+```
+
+- List PowerShell Adapted DSC resources
+
+```powershell
+dsc resource list --adapter Microsoft.DSC/PowerShell
+```
+
+- Find DSC resources
+
+```powershell
+dsc resource list '*DSC*'
+```
+
```

Comment:
We should also discuss searching for PSDSC resources specifically, highlighting that they use the `<ModuleName>/<ResourceName>` convention in the search
``````suggestion

- Find specific adapted PSDSC resources

```powershell
dsc resource list --adapter Microsoft.DSC/PowerShell PSDscResources/*
``````